io.github.jazztickets.openflap.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.github.jazztickets.openflap
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: openflap
+rename-desktop-file: openflap.desktop
+rename-icon: openflap
+rename-appdata-file: openflap.xml
+
+finish-args:
+  - --device=all
```

Comment:
I assume for access to joysticks, controllers etc. correct?

Reply:
correct